src/gui/guiScrollContainer.h

```diff
@@ -56,7 +57,8 @@ class GUIScrollContainer : public gui::IGUIElement
 
 	GUIScrollBar *m_scrollbar;
 	OrientationEnum m_orientation;
-	f32 m_scrollfactor;
+	f32 m_scrollfactor; //< abs(scollfactor * scrollstep) = container pixels to scroll
```

Comment:
What do you mean by "scrollstep" here?

Reply:
When you change the scrollbar value, either by mouse wheel or using the up/down buttons.
```Lua
class GUIScrollBar

	s32 getLargeStep() const { return large_step; }
	s32 getSmallStep() const { return small_step; }
```
What would be a more intuitive name?

Reply:
That's what I expected, but I didn't understand what you want to document here. For documenting m_scrollfactor, I wouldn't mention these "step" values at all:

```suggestion
	f32 m_scrollfactor; //< scrollbar pos * scrollfactor = scroll offset in pixels
```

Reply:
I see where the discrepancy comes from. By scrolling, I think of moving the container by a delta (i.e. steps) of some pixels, which is proportional to `m_scrollfactor`. This of course also holds true for absolute "set pos"-like scroll positions. Both are correct; just different ways to look at it.